portfolio/src/main/webapp/blog_script.js

```diff
@@ -15,13 +15,19 @@
 
 function getComments(){
     fetch('/comments').then(response => response.json()).then((comments) =>{
-        const commentHistory = document.getElementById('oldComments');
-        comments.forEach((line) => {
-            commentHistory.appendChild(createCommentElement(line));
-        })
+        const numCommentSections = document.getElementsByClassName('oldComments').length;
+        for(var postId=1; postId<=numCommentSections; postId++){
+            const commentHistory = document.getElementById(postId);
+            comments.forEach((comment) => {
+                if(postId== comment.postId){
+                    commentHistory.appendChild(createCommentElement(comment));
+                }
+            })
+        }
     });
```

Comment:
How about
```
comments.forEach((comment) => {
    const commentHistory = document.getElementById(comment.postId);    
    commentHistory.appendChild(createCommentElement(comment));
})
```

Reply:
That is so much more straightforward. Than you, done.

Reply:
That is so much more straightforward. Thank you and done.

Reply:
I tried this but there is an issue with how it ends up accessing the new posts. It doesn't retrieve the posts in the data store.